experiments/prediction/figure1.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+
+from RlGlue import RlGlue
+from utils.Collector import Collector
+from utils.policies import actionArrayToPolicy
+from utils.rl_glue import RlGlueCompatWrapper
+from utils.errors import buildRMSPBE
+
+from environments.RandomWalk import RandomWalk, Tabular, Dependent, Inverted
```

Comment:
This seems like random walk and tabular and dependend and inverted are different environments.